app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Users index</h1>
\ No newline at end of file
+<h1>Users index</h1>
+
+<section>
+<% @users.each do |user| %>
```

Comment:
controller `#index` action renders all users to it's corresponding Views index page ✔